tests/unit/test_context.py

```diff
@@ -391,28 +391,25 @@ def test_aggregation_adding():
     assert c.schema[c.schema_name].function_lists[1].aggregation
 
 
-# TODO: Alter schema is not yet implemented
-# def test_alter_schema(c):
-#     c.create_schema("test_schema")
-#     c.sql("ALTER SCHEMA test_schema RENAME TO prod_schema")
-#     assert "prod_schema" in c.schema
+def test_alter_schema(c):
+    c.create_schema("test_schema")
+    c.sql("ALTER SCHEMA test_schema RENAME TO prod_schema")
+    assert "prod_schema" in c.schema
 
-#     with pytest.raises(KeyError):
-#         c.sql("ALTER SCHEMA MARVEL RENAME TO DC")
+    with pytest.raises(KeyError):
+        c.sql("ALTER SCHEMA MARVEL RENAME TO DC")
 
-#     del c.schema["prod_schema"]
+    del c.schema["prod_schema"]
 
 
-# TODO: Alter table is not yet implemented
-# def test_alter_table(c, df_simple):
-#     c.create_table("maths", df_simple)
-#     c.sql("ALTER TABLE maths RENAME TO physics")
-#     assert "physics" in c.schema[c.schema_name].tables
+def test_alter_table(c, df_simple):
+    c.create_table("maths", df_simple)
+    c.sql("ALTER TABLE maths RENAME TO physics")
+    assert "physics" in c.schema[c.schema_name].tables
```

Comment:
Do we also want to assert here than `maths` not in the tables list?

Reply:
Yeah that makes sense, can make a similar alteration to `test_alter_schema`